cosmwasm/contracts/ibc-translator/src/reply.rs

```diff
@@ -0,0 +1,175 @@
+use crate::{
+    msg::GatewayIbcTokenBridgePayload,
+    state::{CHAIN_TO_CHANNEL_MAP, CURRENT_TRANSFER, CW_DENOMS},
+};
+#[cfg(not(feature = "library"))]
+use anybuf::Anybuf;
```

Comment:
Why do we need to gate `anybuf` with a feature flag?

Reply:
thanks, those shouldn't be there